### PR TITLE
Document Dockerfiles

### PR DESCRIPTION
**Pull request by aludvik**
This PR depends on PR 273

### CODE REVIEW

File: ci/sawtooth-build-debs [hunk 1]
```diff
@@ -13,9 +13,17 @@
 # limitations under the License.
 # ------------------------------------------------------------------------------
 
-# USAGE:
-#   From sawtooth-core:
+# Description:
+#   Builds the environment needed to create the Sawtooth Lake python3 debs.
+#   Running the image will put the Sawtooth Lake debs in
+#   sawtooth-core/build/debs.
```
Comment (rbuysse) [inline ci/sawtooth-build-debs]: Amend this to read "Running the image will put the Sawtooth Lake debs in sawtooth-core/build/debs on your local machine."

File: ci/sawtooth-build-docs [hunk 2]
```diff
@@ -13,6 +13,19 @@
 # limitations under the License.
 # ------------------------------------------------------------------------------
 
+# Description:
+#   Builds the environment needed to build the Sawtooth Lake docs
+#   Running the image will put the Sawtooth Lake docs in
+#   sawtooth-core/docs/build
```
Comment (rbuysse) [inline ci/sawtooth-build-docs]: Amend this to read "Running the image will put the Sawtooth Lake docd in sawtooth-core/docs/build on your local machine."

**Comment by aludvik** [comment]
Fixed wording and signed off commit

File: docker/README.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+Sawtooth Lake - Docker
```
Comment (peterschwarz) [inline docker/README.md]: This should be in the Sawtooth 0.8 documentation somewhere, not a README, perhaps as part of a contributors guide.

Comment (rbuysse) [inline docker/README.md]: It's substantially included under the developers guide in the docker documentation updates I have a PR for. https://github.com/hyperledger/sawtooth-core/pull/283

**Pull request merged**